src/schema/metadata/Instructions.yaml

```
@@ -0,0 +1,7 @@
+---
+name: Instructions
+description: |
+  Text of the instructions given to participants before the scan.
+  This is especially important in context of resting state fMRI and
+  distinguishing between eyes open and eyes closed paradigms.
```

Can we get rid of the fMRI here? We can have resting state recordings with all kinds of modalities and the documentation is equally important everywhere.

```suggestion
  Text of the instructions given to participants before the recording.
  This is especially important in context of resting state recordings and
  distinguishing between eyes open and eyes closed paradigms.
```